src/Client.php

```diff
@@ -881,6 +881,7 @@ public function createDeploymentConfig(string $name, string $image_stream_tag, s
       'kind' => 'DeploymentConfig',
       'metadata' => [
         'name' => $name,
+        'labels' => isset($data['labels']) ? $data['labels'] : [ 'name' => $name ],
```

Comment:
Actually, this line results in a php WARNING if labels is unset. You should use http://php.net/array-key-exists